cpp/include/cudf/strings/string_view.cuh

```diff
@@ -330,35 +330,40 @@ __device__ inline size_type string_view::find(const string_view& str,
   return find(str.data(), str.size_bytes(), pos, count);
 }
 
-__device__ inline size_type string_view::find(const char* str,
-                                              size_type bytes,
-                                              size_type pos,
-                                              size_type count) const
-{
-  const char* sptr = data();
-  if (!str || !bytes) return -1;
-  size_type nchars = length();
+template <bool reverse>
```

Comment:
Is there any reasoning about why the template argument here is `reverse` instead of `forward`? I feel a bit weird to call `find<false>()` to find forward.

Reply:
I agree. I will change it to `forward`.